code/game/turfs/simulated/floor/misc_floor.dm

```diff
@@ -163,3 +163,9 @@
 /turf/open/floor/black
 	name = "black floor"
 	icon_state = "black"
+
+/turf/open/floor/plastic
+	name = "plastic floor"
+	desc = "Cheap, lightweight flooring."
```

Comment:
Would be cool if they were actually weaker to reflect this, like the inverse of reinforced plating